eth/stagedsync/stage_headers.go

```diff
@@ -27,6 +28,7 @@ import (
 type HeadersCfg struct {
 	db                kv.RwDB
 	hd                *headerdownload.HeaderDownload
+	statusCh          chan privateapi.ExecutionStatus
```

Comment:
I don't see any code listening on `statusCh`

Reply:
We listen to StatusCh in the ethbackend I believe, so we only need to pass in the current status of the chain when unwinding, which is "SYNCING".

Reply:
Ah, I see.